### ISSUE TITLE
Allow for custom content while using items

### ISSUE THREAD

**Issue by lino-levan** [prabhuignoto/react-chrono#260]
**Is your feature request related to a problem? Please describe.**
Right now, users have to pick between custom jsx content or having metadata. This isn't optimal because we lose some of the cool features of the library otherwise. 

**Describe the solution you'd like**
I would hope for an additional property on `TimelineItemModel` called `content` that would just accept some react element.

Ex:
```jsx
{
    title: "May 1940",
    content: (
        <h1>hi!</h1>
    )
}
```

**Describe alternatives you've considered**
Alternatively, maybe making it possible to customize the metadata inside of JSX could work as well, but this feels less clean.

Ex:
```jsx
<Chrono>
    <div>
        <ChronoMetadata title="May 1940">
        {/*OR*/}
        <ChronoTitle>May 1940</ChronoTitle>
        <h1>hi!</h1>
    </div>
</Chrono>
```

**Additional context**
It particularly becomes an issue without the titles in the timeline (this is not a good example of custom content obviously, but this is just an example):

With Metadata
<img width="1303" alt="Screen Shot 2022-06-21 at 8 38 35 AM" src="https://user-images.githubusercontent.com/11367844/174840983-de6045b8-d614-496a-a65f-80670ba96a38.png">

Without Metadata (No date on timeline)
<img width="1350" alt="Screen Shot 2022-06-21 at 8 39 52 AM" src="https://user-images.githubusercontent.com/11367844/174840990-6eff260a-638b-4da8-b164-de4acc23b3ca.png">

**Comment by jnachtigall** [comment]
Good idea. This would also fix #298

**Comment by prabhuignoto** [comment]
@jnachtigall @lino-levan this is now possible via the new `timelineContent` property of each timeline object. please note `timelineContent` has higher precedence over `cardDetailedText`

Example:

const items = [
  { title: 'Timeline title 1', cardTitle: 'Card Title 1' },
  { title: 'Timeline title 2', cardTitle: 'Card Title 2', timelineContent: <span>Test</span> },
];

<Chrono mode="VERTICAL" items={items}>
</Chrono>;


[Timeline item properties](https://github.com/prabhuignoto/react-chrono#timeline-item-model)
